src/lidar/bg_flood_model.py

```diff
@@ -132,13 +134,7 @@ def main():
     instruction_file = "src/lidar/instructions_bgflood.json"
     with open(instruction_file, "r") as file_pointer:
         instructions = json.load(file_pointer)
-    cache_path = pathlib.Path(instructions["instructions"]["data_paths"]["local_cache"])
-    catchment_boundary_path = (
-        cache_path
-        / instructions["instructions"]["data_paths"]["subfolder"]
-        / instructions["instructions"]["data_paths"]["catchment_boundary"]
-    )
-    catchment_boundary = gpd.read_file(catchment_boundary_path)
+    catchment_boundary = dem_metadata_in_db.get_catchment_boundary(instructions)
     resolution = instructions["instructions"]["output"]["grid_params"]["resolution"]
     # Saving the outputs after each 100 seconds
     outputtimestep = 100.0
```

Comment:
It looks like the comment below this does not match with the code

Reply:
It looks like it has never been right since it was first commited. I've added my own commit to fix it

Reply:
Good suggestion